awx/ui_next/src/screens/WorkflowApproval/shared/WorkflowApprovalStatus.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { Label, Tooltip } from '@patternfly/react-core';
+import { CheckIcon, InfoCircleIcon } from '@patternfly/react-icons';
+import { WorkflowApproval } from '../../../types';
+import { formatDateString } from '../../../util/dates';
+
+function WorkflowApprovalStatus({ workflowApproval, i18n }) {
```

Comment:
We have a generic status label component at `components/StatusLabel`. Do you think it could meet your needs here instead of a new component?

Reply:
👍 will check that out

Reply:
Yea there's definitely some overlap here.  If we want to keep the same UX then I'd have to refactor this component quite a bit with several conditionals.  I'm a little reluctant to do that given how clean this component is.